{"name": "test_login_with_invalid_email_and_invalid_password", "status": "passed", "start": 1700241415506, "stop": 1700241417728, "uuid": "26f9f0b4-a153-49dc-98de-cb070d9c1df6", "historyId": "d3edc9f72c3d0d39ccf9951c28c59331", "testCaseId": "d3edc9f72c3d0d39ccf9951c28c59331", "fullName": "Tests.test_login.TestLogin#test_login_with_invalid_email_and_invalid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-6QNF2MS"}, {"name": "thread", "value": "17304-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_login"}]}